### PR TITLE
Report breaches once per item in delete_confirmation_info

### PR DESCRIPTION
**Pull request by jaroel**
You'd get multiple entries for one page if there were multiple blocks w/ links on a page.

### CODE REVIEW

**Review by sneridagh — APPROVED** [review]

File: plone/app/linkintegrity/browser/info.py [hunk 1]
```diff
@@ -101,7 +102,23 @@ def get_breaches(self, items=None):
...
+        for result in results:
+            target_uid = result["target"]["uid"]
+            uid_target[target_uid] = result["target"]
+            sources = uid_sources[target_uid]
+            for source in result["sources"]:
+                if source not in sources:
+                    sources.append(source)
+
+        # List of breaches
+        return [
```
Comment (tisto) [inline plone/app/linkintegrity/browser/info.py:118]: @jaroel @mauritsvanrees I guess this is a breaking change in plone.app.querystring, right? So we have to do a major version dump and include it (only) in Plone 6.1, am I correct?

Comment (jaroel) [reply plone/app/linkintegrity/browser/info.py:118]: Nah, this doesn't change the shape - just de-duplication.
I'll add a test to make this more visible, fair enough.

**Comment by tisto** [comment]
@jaroel would it be possible to group all breaches on a single page on the frontend? Or is this something that we need to take into consideration on the backend already?

**Comment by jaroel** [comment]
@tisto not a problem; looks like the screenshot in https://github.com/plone/volto/pull/5234 ;)

**Comment by tisto** [comment]
@jaroel @sneridagh we might have a problem here. p.a.linkintegrity dropped Plone 5.2 compatibility with version 4.0.0:

https://github.com/plone/plone.app.linkintegrity/blob/master/CHANGES.rst#400-2023-03-13

For FZJ we need Plone 5.2 support. This means we have to backport this feature to p.a.linkintegrity 3.x.x if I am not mistaken.

**Comment by mauritsvanrees** [comment]
It is too late in the evening, I can't wrap my brain around this change.  Too many targets with sources and sources with targets. :-/

> For FZJ we need Plone 5.2 support. This means we have to backport this feature to p.a.linkintegrity 3.x.x if I am not mistaken.

I would not call it a feature, but a bugfix.  But we can disagree.

If we do not back port this, I expect you may get a few duplicates.  That seems not so bad, but would be nice to fix.  Or does Volto or restapi give an error when there are duplicates?  That would make it more urgent.

File: CHANGES.rst [hunk 2]
```diff
@@ -17,6 +17,9 @@ Bug fixes:
 - Remove outgoing 'isReferencing' RelationValues from catalog on deleting content item.
   [ksuess] (#93)
 
+- Report breaches once per item in delete_confirmation_info
```
Comment (mauritsvanrees) [inline CHANGES.rst:20]: You added this under an already released version.
Please put this in a new file instead, named something like `news/95.bugfix`.

**Comment by tisto** [comment]
@mauritsvanrees I am ok with it either way. I am also thorn between bugfix (technically correct) or feature (because it is nice to have it). Let's call it a bugfix then.

I'd expect Volto being able to work with both versions. We need to abstract away changes in "Plone core" either in plone.restapi or Volto. If we introduce a hard dependency on versions, this means lots of things to explain to people, which I would like to avoid.

I'd like to merge and backport this today then and do releases. @mauritsvanrees do you want to take care of this or shall I help out?

File: CHANGES.rst (hunk 2 above)
Comment (tisto) [reply CHANGES.rst:20]: @jaroel needs to adapt to our new ways. You have been out of the loop for way too long my friend. :)

Comment (jaroel) [reply CHANGES.rst]: I should know better. Thanks!

**Comment by tisto** [comment]
@jenkins-plone-org please run jobs

**Comment by mauritsvanrees** [comment]
> I'd like to merge and backport this today then and do releases. @mauritsvanrees do you want to take care of this or shall I help out?

@tisto  Go ahead.

**Comment by tisto** [comment]
@jaroel I did a release:

https://pypi.org/project/plone.app.linkintegrity/4.0.3/

Could you open another PR that backports this fix to 3.x.x?

**Comment by tisto** [comment]
@jaroel ok, you did this already. :)

**Pull request merged**